F/Fmt/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Fmt"
+version = v"9.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/fmtlib/fmt.git", "a33701196adfad74917046096bf5a2aa0ab0bb50")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+
+cd $WORKSPACE/srcdir/fmt
+mkdir build
+cd build
+
+cmake \
+-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN%.*}_gcc.cmake \
+-DCMAKE_BUILD_TYPE=Release \
+-DBUILD_SHARED_LIBS=True \
+-DCMAKE_INSTALL_PREFIX=${prefix} \
+..
```

Comment:
Don't build the tests
```suggestion
    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
    -DCMAKE_BUILD_TYPE=Release \
    -DBUILD_SHARED_LIBS=True \
    -DFMT_TEST=OFF \
    -DCMAKE_INSTALL_PREFIX=${prefix} \
    ..
```

Also, I'm moderately sure I [already suggested](https://github.com/JuliaPackaging/Yggdrasil/pull/5565#discussion_r979091238) using some indentation before.